src/jobs/aws-logs-check.ts

```diff
@@ -0,0 +1,117 @@
+import '../config/env';
+import { CloudWatchLogs, FilteredLogEvent } from '@aws-sdk/client-cloudwatch-logs';
+import logger from '../config/logger';
+import { discordNotifier } from '../services/discord';
+import { isDevelopment } from '../utils/env';
+
+const JOB_NAME = 'AWS-LOGS';
+const TIME_THRESHOLD_MINUTES = process.env.AWS_LOGS_TIME_THRESHOLD_MINUTES;
```

Comment:
Let's combine all process.env in one place and them import from there. For example some config file.